app/models/paper.rb

```diff
@@ -43,4 +45,8 @@ def valid_methodologies
       'case report'
     ]
   end
+
+  def find_publication
+    self.publication = Publication.where(name: self.publication.name).first_or_initialize
```

Comment:
yea, before validation.  This will let it select an existing publication instead of creating a new one.